logs/logs_test.go

```diff
@@ -79,7 +79,7 @@ func TestApplication(t *testing.T) {
 
 			tc.cmd.out = out
 
-			if err := tc.cmd.Run(ctx, apiClient, "app", "app-name"); err != nil {
+			if err := tc.cmd.Run(ctx, apiClient, ApplicationQuery("app-name", "app-ns")); err != nil {
```

Comment:
missing similar test for `BuildQuery`

Reply:
You are right but I felt like it's sort of pointless as the only thing that changes is the query string. Since we don't have a real loki the query doesn't really do anything.